src/test/java/org/opentripplanner/updater/trip/RealtimeTripInput.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.updater.trip;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.opentripplanner.framework.time.TimeUtils;
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.site.RegularStop;
+
+public record RealtimeTripInput(String id, Route route, List<StopCall> stops) {
```

Comment:
This is used for setting up scheduled trips right? It might be a bit confusing to call it RealtimeTripInput in that case. How about just calling it TripInput?

Reply:
I've renamed it.